### PR TITLE
Update nsv2 action text

### PR DESCRIPTION
**Pull request by uu-z**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change
> /kind ui-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: translations/en.yaml [hunk 1]
```diff
@@ -76,15 +76,15 @@ action.type.startSubChain: 'start SubChain'
...
-action.type.stakeCreate: 'Stake Create'
-action.type.stakeUnstake: 'Stake Unstake'
-action.type.stakeWithdraw: 'Stake Withdraw'
-action.type.stakeAddDeposit: 'Stake Add Deposit'
-action.type.stakeRestake: 'Stake Restake'
-action.type.stakeChangeCandidate: 'Stake Change Candidate'
-action.type.stakeTransferOwnership: 'Stake Transfer Ownership'
-action.type.candidateRegister: 'Candidate Register'
-action.type.candidateUpdate: 'Candidate Update'
+action.type.stakeCreate: 'Create Stask'
```
Comment (guo) [inline translations/en.yaml:79]: typo.

File: translations/en.yaml [hunk 2]
```diff
@@ -76,15 +76,15 @@ action.type.startSubChain: 'start SubChain'
...
+action.type.stakeUnstake: 'Unstake'
+action.type.stakeWithdraw: 'Withdraw Stake'
+action.type.stakeAddDeposit: 'Deposit to Stake'
+action.type.stakeRestake: 'Restake'
+action.type.stakeChangeCandidate: 'Change Candidate'
+action.type.stakeTransferOwnership: 'Transfer Stake'
+action.type.candidateRegister: 'Register Candidate'
```
Comment (guo) [inline translations/en.yaml:86]: let's keep the old one

File: translations/en.yaml [hunk 3]
```diff
@@ -76,15 +76,15 @@ action.type.startSubChain: 'start SubChain'
...
+action.type.candidateUpdate: 'Update Candidate'
```
Comment (guo) [inline translations/en.yaml:87]: keep the old one.

**Comment by guo** [comment]
please update the text before merge.

**Comment by uu-z** [comment]
updated

**Pull request merged**